beta/kustomize/README.md

```diff
@@ -0,0 +1,188 @@
+- [Appendix: Customize your deployment](#appendix-customize-your-deployment)
+
+## Before you begin
+
+Before you begin, familiarize yourself with [PREPARATION.md](/PREPARATION.md) and complete the necessary steps there.
+
+### In this folder
+
+- [`/base`](./base/): The default deployment that the overlays build on top of to
+deploy your preferred bridge.
+- [`/overlays`](./overlays/): The overlays to customize your deployment.
+
+## Step 1: Prepare your `scimsession` file
+
+After downloading your `scimsession` file after creating your SCIM bridge credentials,
+place it in the folder `./base`. The path should be `./base/scimsession`. Kubernetes
+will use this file along with a secret generator to create the Kubernetes secret
+containing the contents of your `scimsession` file, named as `scimsession`.
+
+## Step 2: Determine which overlays to use
+
+There are currently 3 overlays defined for changing  the bridge as well as a deployment
+overlay. In most standard deployments, the `letsencrypt` overlay is going to be used.
+This overlay configures the SCIM bridge to request a certificate from Let's Encrypt using
+a domain name defined in the overlay ConfigMap. If you're bringing your own 
+TLS certificate, the `self-managed-tls` overlay is to be used instead.
+
+If you're using Google Workspace, you'll also need the `workspace` overlay configured
+as documented in that overlay's README. 
+
+## Step 3: Configure the overlays
+
+### Let's Encrypt
+
+To configure the Let's Encrypt overlay, edit the file `patch-configmap.yaml`. Update
+`scim.example.com` to be the domain A record that is pointed at your bridge.
+
+### Self Managed TLS
+
+To bring your own TLS certificate for use with the SCIM bridge, add your TLS private
+key and certificate to the folder `overlays/self-managed-tls` named `tls.key` and
+`tls.cert` respectively.
+
+### Google Workspace
+
+Place the credentials file in the `overlays/workspace` folder. The kustomization overlay will create this as an additional secret.
+
+Edit the file `workspace-settings.json` and fill in correct values for:
+
+- **Actor**: The email address of the administrator in Google Workspace that the service account is acting on behalf of.
+- **Bridge Address**: The URL you will use for your SCIM bridge (not your 1Password account sign-in address). This is most often a subdomain of your choosing on a domain you own. If you're using the Let's Encrypt overlay, this would be the value of `OP_TLS_DOMAIN` configured in `patch-configmap.yaml`
+
+## Step 4: Prepare the deployment
+
+To configure the deployment for use, edit the file
+`overlays/deployment/kustomization.yaml` and make changes as appropriate. For
+example: if you're going to be using a self-managed TLS certificate, then comment out
+the `letsencrypt` overlay and instead uncomment the `self-managed-tls` overlay.
+
+Run the following command to deploy 1Password SCIM Bridge:
+
+```bash
```

Comment:
following the instructions exactly as written, I got an error 
`Error from server (NotFound): error when creating "overlays/deployment/": namespaces "op-scim" not found`

I don't believe the docs instruct a user to create the `op-scim` namespace, and the manifest appears to not do this on behalf of the user. 

Easiest path might be to provide the `kubectl create namespace` command in the docs. 

Alternatively we attempt to create it for them in the manifest before the other resource.